### PR TITLE
Remove all picker imports ahead of rn 0.66 uplift

### PR DESCRIPTION
**Pull request by glenn-axsy**
This change removes various unused imports, including unused Picker and PickerIOS imports. This needs to be done before the upcoming update of RN to version 0.66 as Picker and PickerIOS are no longer imported from react-native.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,5 +33,5 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "version": "0.9.27"
+  "version": "0.9.28"
```
Comment (gillpeacegood) [inline package.json:36]: I suppose this library published alright? (I can't see a publishConfig section)

**Review by gillpeacegood — APPROVED** [review]
Might be worth considering setting up lint to disallow unused imports

File: package.json (hunk 1 above)
Comment (glenn-axsy) [reply package.json]: I published it today, now at version 0.9.29 as 0.9.28 had been taken by recent changes to the lib made by Scott.

**Comment by glenn-axsy** [comment]
> Might be worth considering setting up lint to disallow unused imports

I've not set up lint before, and as it's a fork of 3rd party lib I suspect there would lots of other lint errors to fix, so I've settled for doing a manual sweep through the code to remove unused imports from source files that I hadn't had to change earlier.

**Review by gillpeacegood — APPROVED** [review]

**Pull request merged**